MalwareAnalyzer/views.py

```diff
@@ -79,6 +79,39 @@ def apkid_analysis(app_dir):
         else:
             apkid_dict["manipulator"] = ""
 
+        apkid_dict["result"] = ""

```

Comment:
From here it's double work. @superpoussin22 It should like do scan of APK first, if it fails to do so try dex scan and not both.

Reply:
@ajinabraham for the moment scanning only the apk did not provide all the informations, need to check apid.py to undestand why , so far it's a quick fix

Reply:
okey then